src/v3/src/components/hocs/withFormValidationState.tsx

```diff
@@ -84,7 +84,9 @@ export const withFormValidationState: WrappedFunctionComponent<
       setTouched(false);
     }, [messages?.value]);
 
-    const fieldErrors = touched ? errors : errorsFromSchema;
+    // Server side message should not disappear during client validation
+    // It should only be updated with data from the next transaction
+    const fieldErrors = !!errorsFromSchema ? errorsFromSchema : errors;
```

Comment:
Let's discuss this tomorrow -- I'm not sure that the issue here is obviously an issue so we should talk about it before we change behavior

Reply:
Based on discussion, now it's only following validation from form submission.